### ISSUE TITLE
Error during `mix deps.compile` under Windows 10

### ISSUE THREAD

**Issue by jakubpawlowicz** [bake-bake-bake/bakeware#115]
Hey all,

Sorry if that's a noob question, but what's the environment I should run Windows builds under?
Is it cmd.exe, Powershell, or a linux shell under WSL?

I have tried the first two, and what I get (under Windows 10) is:
```
mkdir -p c:\Users\kubap\Desktop\wxtest\_build\dev\lib\bakeware\obj
The syntax of the command is incorrect
```

I set everything up as described in https://github.com/bake-bake-bake/bakeware#building-on-windows and looked at GH Actions for clues, but no luck so far under Windows.

**Comment by simonmcconnell** [comment]
They're for the msys shell.

I've got a branch I haven't put a PR in for yet which expands the readme a little for windows.

https://github.com/simonmcconnell/bakeware

**Comment by jjcarstens** [comment]
resolved with #116

**Comment by jakubpawlowicz** [comment]
This was fast @simonmcconnell, thanks a ton for your wizardry! I'll give it a try tomorrow.

**Comment by jjcarstens** [comment]
Released in v0.2.1
